modules/apps/segments/segments-web/package.json

```diff
@@ -65,6 +75,7 @@
 	"private": true,
 	"scripts": {
 		"build": "babel --source-maps -d classes/META-INF/resources src/main/resources/META-INF/resources && liferay-npm-bundler",
+		"start": "cpr ../../../../modules/apps/frontend-theme/frontend-theme-admin/build/images/lexicon dev-build/lexicon -d && cpr src/main/resources/META-INF/resources/assets dev-build/assets -d && webpack-dev-server --config webpack.dev.config.js",
```

Comment:
I'm not positive about this... Seems a little strange to be copying files on start. I'd rather point to the original location of these files via webpack, otherwise the hot-reloading won't apply to these files.